web/models.py

```
@@ -25,6 +25,67 @@
 from web.utils import calculate_and_update_user_streak
 
 
+class Campaign(models.Model):
+    """
+    Model representing a crowdfunding campaign created by a teacher.
+    """
+
+    teacher = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="campaigns", help_text="Teacher who created the campaign."
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)